submissions/expero1/html-css-popup/style.css

```diff
@@ -0,0 +1,405 @@
+	display: block;
+position:relative;
+/*top:-8px;*/
+	margin: 10px 0 5px 0;
+	height: 1px;
+	border-bottom: 1px solid gray;
+	box-shadow: 0 0 10px gray;
+}
+.show-more-checkbox:checked ~ .popup-menu-inner.hidden {
+	display: grid;
+}
+.show-more-checkbox:focus ~ .show-more-btn {
+background-color: gray;
+}
+
+.show-more-checkbox:checked ~ .even-more-btn {
+	display: block;
+}
+
+.even-more-btn {
+	width: 100%;
+	display: none;
+margin-top: 10px;
+
+	border: none;
+	cursor: pointer;
+	height: 3em;
+}
+.even-more-btn:hover,
+.even-more-btn:focus {
+	background-color: gray;
+}
+
+.notification-menu {
+	grid-area: notification-menu;
+}
+.user-menu{
+	grid-area:user-menu;
+}
+.user-menu img,
+.notification-menu img {
+	object-fit: cover;
+	width: 30px;
+}
+
+main {
+	height: 110vh;
+	background-color: Gainsboro;
+	z-index: -1;
+	position: relative;
+}
+main::after {
+	content: 'main section';
+	color:white;
+	position: absolute;
+	margin: 0 auto;
+	top: 50vh;
+	right: 50%;
+	text-transform: uppercase;
+	z-index: -1;
+	display: block;
+	text-align: center;
+	transform: translateX(50%);
+	font-size: 50px;
+}
+
+footer {
+	height: 200px;
+	background-color: black;
+	position: relative;
+	z-index: -1;
+	font-size: 30px;
+	text-transform: uppercase;
+}
+footer::after {
+	content: 'Footer';
+	color:white;
+	position: absolute;
+	margin: 0 auto;
+	top: 50%;
+	right: 50%;
+	z-index: -1;
+	transform: translateX(50%);
+}
+
+@media (max-width:800px) {
+	.header-wrapper{ max-width: 80%;}
+}
+
+/*Smartphone portrait mode*/
+@media (max-width:550px) {
+	header {
+		height: 100px;
+		position: fixed;
+		top:0;
+		left: 0;
+		right: 0;
+	}
+	.header-wrapper {
+		padding-top:10px;
+		padding-bottom: 10px;
+		padding-left: 10px;
+		padding-right: 10px;
+		grid-template-columns: repeat(8,1fr);
+		gap: 10px;
+		grid-template-areas: "logo popup-menu-wrapper"
+													"notification-menu user-menu ";
+		max-width: 100%;
+	}
+	.logo {
+		grid-column: 1/2;
+		grid-row: 1;
+	}
+	.main-menu {
+		grid-column: 1/9;
+		grid-row: 2;
+		border-top: 1px solid gray;
+		padding-top: 10px;
+	}
+	.main-menu ul {
+		gap: 20px;
+		justify-content: center;
+	}
+	.popup-menu-wrapper {
+		grid-column: 6/7;
+		grid-row: 1;
+	}
+	.popup-menu {
+		position: fixed;
+		top: 120px;
+		left:0;
+		right:0;
+		width:90%;
+		min-height: 50px;
+		max-height: 80vh;
+		overflow-x: auto;
+		border: 1px solid grey;
+		border-radius: 5px;
+		box-shadow: 0 0 5px grey;
+		margin: 0 auto;
+	}
+	.user-menu {
+		grid-column: 8/9;
+		grid-row: 1;
+	}
+	.notification-menu {
+		grid-column: 7/8;
+		grid-row: 1;
+	}
+}
+/* Smartphone Landscape Mode */
+@media (max-height:700px) and (max-width:900px) and (min-width:550px) {
+	.popup-menu {
+		position: absolute;
+		width:450px;
+		/*min-height: 250px;*/
+		top: 60px;
+		right:-150px;
+	}
+
+	.popup-menu-inner {
+		grid-template-columns: repeat(4, 1fr);
+		grid-template-rows: auto;
+		gap:5px;
+	}
+	.popup-menu-inner li a {
```

Comment:
Avoid using too specific CSS selectors. [Read more](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)

Reply:
Thanks. Fixed it.